app/views/layouts/application.html.erb

```diff
@@ -23,7 +23,8 @@
       </div>
       <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
         <ul class="nav navbar-nav">
-          <li class="active"><%= link_to "#{icon "server"} Servers".html_safe, root_path %></li>
+          <li class="<%= "active" if controller_name != "users" %>"><%= link_to "#{icon "server"} Servers".html_safe, root_path %></li>
```

Comment:
I think we should think of a better way of making an item active. Only checking if something is the users controller or not is not sufficient.